tests/tests-ci.sh

```diff
@@ -21,9 +21,12 @@ make -j$(nproc)
 set -e
 
 mkdir -p linuxdeployqt.AppDir/usr/{bin,lib}
-cp /usr/bin/{patchelf,desktop-file-validate} /usr/local/bin/{appimagetool,zsyncmake} linuxdeployqt.AppDir/usr/bin/
+cp /usr/bin/{patchelf,desktop-file-validate} linuxdeployqt.AppDir/usr/bin/
 cp ./bin/linuxdeployqt linuxdeployqt.AppDir/usr/bin/
-cp -r /usr/local/lib/appimagekit linuxdeployqt.AppDir/usr/lib/
+cp -R /tmp/appimagekit.AppDir linuxdeployqt.AppDir/usr/appimagekit
```

Comment:
Where is `/tmp/appimagekit.AppDir` supposed to be coming from? I can't seem to see that being created...

Reply:
Check the other script.